packages/cli/src/metadataGeneration/typeResolver.ts

```diff
@@ -1109,6 +1110,10 @@ export class TypeResolver {
       return undefined;
     }
   }
+
+  private getNodeReadOnly(node: ts.PropertySignature) {
+    return node.modifiers?.some(modifier => modifier.kind === ts.SyntaxKind.ReadonlyKeyword) || isExistJSDocTag(node, tag => tag.tagName.text === 'readonly');
```

Comment:
```suggestion
    return node.modifiers?.some(modifier => modifier.kind === ts.SyntaxKind.ReadonlyKeyword);
```

Reply:
Hi @WoH Thanks for your reply! I will look into your suggestions this evening or over the weekend.

In regards to your comment:
> Also, I think 2 ways are confusing, if we can avoid them. Let's stick to the TS keyword unless there's a good use case.

My reasoning is that the keyword `readonly` makes the compiler look for illegal property assignments, this can be a breaking change in case someone just want to make use of the swagger `readOnly` option that simply includes the property in a response but not in a request. Let me know what you think though.

Reply:
I agree that this change would be breaking.